std/array.d

```diff
@@ -3774,6 +3778,70 @@ if (isDynamicArray!A)
     assert(app2[] == [ 1, 2, 3, 4, 5, 6 ]);
 }
 
+//https://github.com/dlang/phobos/pull/8789 issue for InPLaceAppenders
+@safe pure nothrow unittest
+{
+    import std.array : appender;
+
+    // Append characters one by one
+    auto a = appender!string();
+    foreach (c; "Dlang")
+        a.put(c);
+    assert(a[] == "Dlang");
+}
+
+@safe pure nothrow unittest
+{
+    import std.array : appender;
+
+    // Append full strings
+    auto a = appender!string();
+    a.put("Hello");
+    a.put(", ");
+    a.put("world!");
+    assert(a[] == "Hello, world!");
+}
+
+
+
```

Comment:
Please trim this to a single empty line between the two unittests. (No need for three of them.)